src/catalog/src/builtin.rs

```diff
@@ -4054,6 +4071,30 @@ WHERE worker_id = 0",
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_DATAFLOW_GLOBAL_IDS: LazyLock<BuiltinView> = LazyLock::new(|| BuiltinView {
```

Comment:
We should leave a note somewhere how this is different from [`mz_compute_exports`](https://materialize.com/docs/sql/system-catalog/mz_introspection/#mz_compute_exports).

Reply:
Will record this in the docs I add (per your other comment). We should probably collect somewhere all of the notions of 'id' that occur in MZ, explain what they are, and ensure we use the correct names for things.